src/fsharp/build.fs

```diff
@@ -2031,9 +2036,11 @@ type TcConfigBuilder =
       /// if true - every expression in quotations will be augmented with full debug info (filename, location in file)
       mutable emitDebugInfoInQuotations : bool
 
+#if FX_SHADOWCOPY_IN_FSI
```

Comment:
Is this any different that `FX_NO_APPDOMAINS`? Since app domains is the feature that support shadow copy.